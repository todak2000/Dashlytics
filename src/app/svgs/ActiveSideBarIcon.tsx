"use client";
import React from "react";
import { SvgProps } from "../constants/types";
const ActiveSideBarIcon: React.FC<SvgProps> = ({ className }) => {
  return (
    <div className={className}>
      <svg viewBox="0 0 3 21" fill="none" preserveAspectRatio="xMidYMid meet">
        <path
          d="M6.53467e-06 3.02509C7.11773e-06 1.42129 1.40951 0.182713 3 0.388889V0.388889V21V21C1.34315 21 4.88293e-07 19.6569 1.09063e-06 18L6.53467e-06 3.02509Z"
          fill="#0D062D"
        />
      </svg>
    </div>
  );
};
export default React.memo(ActiveSideBarIcon);
