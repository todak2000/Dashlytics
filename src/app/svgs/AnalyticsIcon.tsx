"use client";
import React from "react";
import { SvgProps } from "../constants/types";
const AnalyticsIcon: React.FC<SvgProps> = ({ className }) => {
  return (
    <div className={className}>
      <svg viewBox="0 0 40 40" fill="none" preserveAspectRatio="xMidYMid meet">
        <path
          d="M10 20.98V23C10 28 12 30 17 30H23C28 30 30 28 30 23V17C30 12 28 10 23 10H17C12 10 10 12 10 17"
          stroke="currentColor"
          strokeWidth="1.5"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M19.17 20L18.7 19.3L15.5 22.5"
          stroke="currentColor"
          strokeWidth="1.5"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M24.5 17.5L22.99 19.01L22 20"
          stroke="currentColor"
          strokeWidth="1.5"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M22.5 17.5H24.5V19.5"
          stroke="currentColor"
          strokeWidth="1.5"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </div>
  );
};
export default React.memo(AnalyticsIcon);
