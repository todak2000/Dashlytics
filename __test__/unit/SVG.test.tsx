import { render } from "@testing-library/react";

import ActiveSideBarIcon from "@/app/svgs/ActiveSideBarIcon";
import AnalyticsIcon from "@/app/svgs/AnalyticsIcon";
import BoxIcon from "@/app/svgs/BoxIcon";
import DarkModeIcon from "@/app/svgs/DarkModeIcon";
import DashboardIcon from "@/app/svgs/DashboardIcon";
import InfoIcon from "@/app/svgs/InfoIcon";
import LightModeIcon from "@/app/svgs/LightModeIcon";
import LogoIcon from "@/app/svgs/LogoIcon";
import OtherIcon from "@/app/svgs/OtherIcon";
import PowerIcon from "@/app/svgs/PowerIcon";
import SettingIcon from "@/app/svgs/SettingIcon";
import SettingTwoIcon from "@/app/svgs/SettingTwoIcon";
import UserIcon from "@/app/svgs/UserIcon";

describe("ActiveSideBar Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<ActiveSideBarIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Analytics Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<AnalyticsIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Box Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<BoxIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Darkmode Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<DarkModeIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Dashboard Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<DashboardIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Info Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<InfoIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("LightMode Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<LightModeIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Logo Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<LogoIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Other Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<OtherIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Power Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<PowerIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("Setting Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<SettingIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});
describe("Setting Two Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<SettingTwoIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});
describe("User Icon", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<UserIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});
