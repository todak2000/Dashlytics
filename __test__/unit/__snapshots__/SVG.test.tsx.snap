// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveSideBar Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 3 21"
    >
      <path
        d="M6.53467e-06 3.02509C7.11773e-06 1.42129 1.40951 0.182713 3 0.388889V0.388889V21V21C1.34315 21 4.88293e-07 19.6569 1.09063e-06 18L6.53467e-06 3.02509Z"
        fill="#0D062D"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Analytics Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M10 20.98V23C10 28 12 30 17 30H23C28 30 30 28 30 23V17C30 12 28 10 23 10H17C12 10 10 12 10 17"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M19.17 20L18.7 19.3L15.5 22.5"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M24.5 17.5L22.99 19.01L22 20"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M22.5 17.5H24.5V19.5"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Box Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M11.1699 15.4401L19.9999 20.55L28.7699 15.47"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M20 29.61V20.54"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M10.3901 17.1701C10.3901 15.7901 11.3801 14.1101 12.5901 13.4401L17.9301 10.4701C19.0701 9.84006 20.9401 9.84006 22.0801 10.4701L27.4201 13.4401C28.6301 14.1101 29.6201 15.7901 29.6201 17.1701V22.8201C29.6201 24.2001 28.6301 25.8801 27.4201 26.5501L22.0801 29.5201C20.9401 30.1501 19.0701 30.1501 17.9301 29.5201L12.5901 26.5501C11.3801 25.8801 10.3901 24.2001 10.3901 22.8201V22.0001"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M24.9998 21.24V17.58L18.9297 14.07L17.8798 13.47L15.5098 12.1"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Darkmode Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 30 30"
    >
      <path
        d="M15 4.6875C15.2486 4.6875 15.4871 4.58873 15.6629 4.41291C15.8387 4.2371 15.9375 3.99864 15.9375 3.75V2.8125C15.9375 2.56386 15.8387 2.3254 15.6629 2.14959C15.4871 1.97377 15.2486 1.875 15 1.875C14.7514 1.875 14.5129 1.97377 14.3371 2.14959C14.1613 2.3254 14.0625 2.56386 14.0625 2.8125V3.75C14.0625 3.99864 14.1613 4.2371 14.3371 4.41291C14.5129 4.58873 14.7514 4.6875 15 4.6875Z"
        fill="currentColor"
      />
      <path
        d="M15 25.3125C14.7514 25.3125 14.5129 25.4113 14.3371 25.5871C14.1613 25.7629 14.0625 26.0014 14.0625 26.25V27.1875C14.0625 27.4361 14.1613 27.6746 14.3371 27.8504C14.5129 28.0262 14.7514 28.125 15 28.125C15.2486 28.125 15.4871 28.0262 15.6629 27.8504C15.8387 27.6746 15.9375 27.4361 15.9375 27.1875V26.25C15.9375 26.0014 15.8387 25.7629 15.6629 25.5871C15.4871 25.4113 15.2486 25.3125 15 25.3125Z"
        fill="currentColor"
      />
      <path
        d="M27.1875 14.0625H26.25C26.0014 14.0625 25.7629 14.1613 25.5871 14.3371C25.4113 14.5129 25.3125 14.7514 25.3125 15C25.3125 15.2486 25.4113 15.4871 25.5871 15.6629C25.7629 15.8387 26.0014 15.9375 26.25 15.9375H27.1875C27.4361 15.9375 27.6746 15.8387 27.8504 15.6629C28.0262 15.4871 28.125 15.2486 28.125 15C28.125 14.7514 28.0262 14.5129 27.8504 14.3371C27.6746 14.1613 27.4361 14.0625 27.1875 14.0625Z"
        fill="currentColor"
      />
      <path
        d="M3.75 14.0625H2.8125C2.56386 14.0625 2.3254 14.1613 2.14959 14.3371C1.97377 14.5129 1.875 14.7514 1.875 15C1.875 15.2486 1.97377 15.4871 2.14959 15.6629C2.3254 15.8387 2.56386 15.9375 2.8125 15.9375H3.75C3.99864 15.9375 4.2371 15.8387 4.41291 15.6629C4.58873 15.4871 4.6875 15.2486 4.6875 15C4.6875 14.7514 4.58873 14.5129 4.41291 14.3371C4.2371 14.1613 3.99864 14.0625 3.75 14.0625Z"
        fill="currentColor"
      />
      <path
        d="M22.9549 5.71908L22.2921 6.38189C22.2025 6.46837 22.1311 6.57182 22.082 6.6862C22.0328 6.80058 22.007 6.92359 22.0059 7.04807C22.0048 7.17256 22.0285 7.29601 22.0757 7.41122C22.1228 7.52644 22.1924 7.63111 22.2804 7.71913C22.3685 7.80716 22.4731 7.87677 22.5884 7.92391C22.7036 7.97105 22.827 7.99477 22.9515 7.99369C23.076 7.9926 23.199 7.96674 23.3134 7.91761C23.4278 7.86848 23.5312 7.79705 23.6177 7.70751L24.2814 7.04564C24.3685 6.95854 24.4376 6.85513 24.4848 6.74132C24.5319 6.62752 24.5562 6.50554 24.5562 6.38236C24.5562 6.25917 24.5319 6.1372 24.4848 6.02339C24.4376 5.90959 24.3685 5.80618 24.2814 5.71908C24.1943 5.63197 24.0909 5.56288 23.9771 5.51574C23.8633 5.4686 23.7413 5.44434 23.6182 5.44434C23.495 5.44434 23.373 5.4686 23.2592 5.51574C23.1454 5.56288 23.042 5.63197 22.9549 5.71908Z"
        fill="currentColor"
      />
      <path
        d="M6.38185 22.292L5.7181 22.9548C5.631 23.0419 5.5619 23.1453 5.51476 23.2591C5.46762 23.3729 5.44336 23.4949 5.44336 23.6181C5.44336 23.7413 5.46762 23.8632 5.51476 23.977C5.5619 24.0909 5.631 24.1943 5.7181 24.2814C5.89401 24.4573 6.1326 24.5561 6.38138 24.5561C6.50456 24.5561 6.62654 24.5318 6.74035 24.4847C6.85415 24.4376 6.95756 24.3685 7.04466 24.2814L7.70748 23.6185C7.79458 23.5315 7.86368 23.4282 7.91085 23.3144C7.95801 23.2007 7.98231 23.0787 7.98235 22.9556C7.9824 22.8325 7.95819 22.7105 7.9111 22.5967C7.86402 22.483 7.79499 22.3796 7.70794 22.2925C7.6209 22.2054 7.51756 22.1362 7.40381 22.0891C7.29006 22.0419 7.16813 22.0176 7.04499 22.0176C6.7963 22.0175 6.55776 22.1162 6.38185 22.292Z"
        fill="currentColor"
      />
      <path
        d="M23.6171 22.292C23.4403 22.1212 23.2035 22.0267 22.9577 22.0288C22.7119 22.031 22.4767 22.1296 22.3029 22.3034C22.1291 22.4772 22.0305 22.7124 22.0284 22.9582C22.0262 23.204 22.1207 23.4408 22.2915 23.6176L22.9543 24.2814C23.1302 24.4573 23.3688 24.5561 23.6176 24.5561C23.8664 24.5561 24.1049 24.4573 24.2809 24.2814C24.4568 24.1054 24.5556 23.8668 24.5556 23.6181C24.5556 23.3693 24.4568 23.1307 24.2809 22.9548L23.6171 22.292Z"
        fill="currentColor"
      />
      <path
        d="M6.38185 7.70845C6.55866 7.87922 6.79548 7.97372 7.04129 7.97158C7.2871 7.96945 7.52223 7.87085 7.69605 7.69703C7.86987 7.52321 7.96847 7.28807 7.97061 7.04226C7.97274 6.79646 7.87825 6.55964 7.70747 6.38283L7.04466 5.71908C6.86875 5.54316 6.63016 5.44434 6.38138 5.44434C6.1326 5.44434 5.89401 5.54316 5.7181 5.71908C5.54219 5.89499 5.44336 6.13358 5.44336 6.38236C5.44336 6.63114 5.54219 6.86973 5.7181 7.04564L6.38185 7.70845Z"
        fill="currentColor"
      />
      <path
        d="M23.1567 16.3903C23.0357 16.271 22.8846 16.1867 22.7195 16.1466C22.5543 16.1065 22.3814 16.112 22.2192 16.1625C21.5901 16.3568 20.9351 16.4542 20.2767 16.4513C18.4926 16.4495 16.7821 15.7401 15.5205 14.4787C14.2589 13.2173 13.5493 11.5069 13.5473 9.72285C13.545 9.06465 13.643 8.40995 13.8379 7.78128C13.8888 7.61887 13.8945 7.44566 13.8544 7.28025C13.8144 7.11483 13.73 6.96344 13.6104 6.84232C13.4909 6.7212 13.3406 6.63492 13.1757 6.59273C13.0108 6.55055 12.8375 6.55404 12.6745 6.60285C11.2513 7.03321 9.96401 7.82507 8.9381 8.9012C7.91218 9.97733 7.18269 11.301 6.82077 12.7431C6.45885 14.1851 6.47687 15.6964 6.87306 17.1294C7.26925 18.5624 8.03008 19.8683 9.08136 20.9197C10.1326 21.9711 11.4385 22.732 12.8715 23.1283C14.3045 23.5247 15.8157 23.5428 17.2578 23.181C18.6999 22.8192 20.0236 22.0898 21.0999 21.064C22.1761 20.0382 22.9681 18.751 23.3985 17.3278C23.4473 17.1644 23.4506 16.9907 23.408 16.8255C23.3654 16.6603 23.2785 16.5098 23.1567 16.3903Z"
        fill="currentColor"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Dashboard Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 24 24"
    >
      <path
        d="M7.24 2H5.34C3.15 2 2 3.15 2 5.33V7.23C2 9.41 3.15 10.56 5.33 10.56H7.23C9.41 10.56 10.56 9.41 10.56 7.23V5.33C10.57 3.15 9.42 2 7.24 2Z"
        fill="currentColor"
      />
      <path
        d="M18.6699 2H16.7699C14.5899 2 13.4399 3.15 13.4399 5.33V7.23C13.4399 9.41 14.5899 10.56 16.7699 10.56H18.6699C20.8499 10.56 21.9999 9.41 21.9999 7.23V5.33C21.9999 3.15 20.8499 2 18.6699 2Z"
        fill="currentColor"
        opacity="0.4"
      />
      <path
        d="M18.6699 13.4301H16.7699C14.5899 13.4301 13.4399 14.5801 13.4399 16.7601V18.6601C13.4399 20.8401 14.5899 21.9901 16.7699 21.9901H18.6699C20.8499 21.9901 21.9999 20.8401 21.9999 18.6601V16.7601C21.9999 14.5801 20.8499 13.4301 18.6699 13.4301Z"
        fill="currentColor"
      />
      <path
        d="M7.24 13.4301H5.34C3.15 13.4301 2 14.5801 2 16.7601V18.6601C2 20.8501 3.15 22.0001 5.33 22.0001H7.23C9.41 22.0001 10.56 20.8501 10.56 18.6701V16.7701C10.57 14.5801 9.42 13.4301 7.24 13.4301Z"
        fill="currentColor"
        opacity="0.4"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Info Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M20 16V21"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M12 14C10.75 15.67 10 17.75 10 20C10 25.52 14.48 30 20 30C25.52 30 30 25.52 30 20C30 14.48 25.52 10 20 10C18.57 10 17.2 10.3 15.97 10.85"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M19.9946 24H20.0036"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`LightMode Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 30 30"
    >
      <rect
        fill="currentColor"
        height="30"
        rx="15"
        width="30"
      />
      <g
        clip-path="url(#clip0_1991_690)"
      >
        <path
          d="M15 19.2188C17.33 19.2188 19.2188 17.33 19.2188 15C19.2188 12.67 17.33 10.7812 15 10.7812C12.67 10.7812 10.7812 12.67 10.7812 15C10.7812 17.33 12.67 19.2188 15 19.2188Z"
          fill="white"
        />
        <path
          d="M14.9998 9.09345C14.5334 9.09345 14.156 8.71564 14.156 8.2497V7.40625C14.156 6.94031 14.5334 6.5625 14.9998 6.5625C15.4661 6.5625 15.8435 6.94031 15.8435 7.40625V8.2497C15.8435 8.71564 15.4661 9.09345 14.9998 9.09345Z"
          fill="white"
        />
        <path
          d="M19.7734 11.0706C19.5576 11.0706 19.3417 10.9882 19.1769 10.8234C18.8473 10.4939 18.8473 9.95964 19.1769 9.63015L19.7734 9.03369C20.103 8.7043 20.637 8.7042 20.9666 9.03379C21.2961 9.36328 21.2961 9.89751 20.9666 10.227L20.37 10.8235C20.2052 10.9882 19.9893 11.0706 19.7734 11.0706Z"
          fill="white"
        />
        <path
          d="M22.5937 15.8438H21.75C21.2836 15.8438 20.9062 15.4659 20.9062 15C20.9062 14.5341 21.2836 14.1562 21.75 14.1562H22.5937C23.0601 14.1562 23.4375 14.5341 23.4375 15C23.4375 15.4659 23.0601 15.8438 22.5937 15.8438Z"
          fill="white"
        />
        <path
          d="M20.37 21.2132C20.1541 21.2132 19.9382 21.1309 19.7734 20.9661L19.1769 20.3697C18.8473 20.0402 18.8473 19.5059 19.1769 19.1764C19.5065 18.8469 20.0404 18.8468 20.37 19.1763L20.9666 19.7728C21.2961 20.1023 21.2961 20.6365 20.9666 20.966C20.8018 21.1308 20.5859 21.2132 20.37 21.2132Z"
          fill="white"
        />
        <path
          d="M14.9998 23.4372C14.5334 23.4372 14.156 23.0594 14.156 22.5935V21.75C14.156 21.2841 14.5334 20.9062 14.9998 20.9062C15.4661 20.9062 15.8435 21.2841 15.8435 21.75V22.5935C15.8435 23.0594 15.4661 23.4372 14.9998 23.4372Z"
          fill="white"
        />
        <path
          d="M9.63037 21.2136C9.41449 21.2136 9.19861 21.1312 9.03381 20.9664C8.70422 20.6369 8.70422 20.1027 9.03381 19.7732L9.63037 19.1768C9.95996 18.8474 10.4939 18.8473 10.8235 19.1769C11.1531 19.5063 11.1531 20.0406 10.8235 20.3701L10.2269 20.9665C10.0621 21.1312 9.84625 21.2136 9.63037 21.2136Z"
          fill="white"
        />
        <path
          d="M8.25 15.8438H7.40625C6.93988 15.8438 6.5625 15.4659 6.5625 15C6.5625 14.5341 6.93988 14.1562 7.40625 14.1562H8.25C8.71637 14.1562 9.09375 14.5341 9.09375 15C9.09375 15.4659 8.71637 15.8438 8.25 15.8438Z"
          fill="white"
        />
        <path
          d="M10.2269 11.0706C10.011 11.0706 9.79517 10.9883 9.63037 10.8235L9.03381 10.2271C8.70422 9.89759 8.70422 9.36335 9.03381 9.03386C9.3634 8.70427 9.89734 8.70417 10.2269 9.03376L10.8235 9.63022C11.1531 9.95971 11.1531 10.4939 10.8235 10.8234C10.6587 10.9882 10.4428 11.0706 10.2269 11.0706Z"
          fill="white"
        />
      </g>
      <defs>
        <clippath
          id="clip0_1991_690"
        >
          <rect
            fill="white"
            height="16.875"
            transform="translate(6.5625 6.5625)"
            width="16.875"
          />
        </clippath>
      </defs>
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Logo Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        clip-rule="evenodd"
        d="M20 40C31.0457 40 40 31.0457 40 20C40 8.95431 31.0457 0 20 0C8.95429 0 0 8.95431 0 20C0 31.0457 8.95429 40 20 40ZM26.2393 9.31684C26.543 8.23805 25.4961 7.60013 24.54 8.2813L11.1931 17.7896C10.1562 18.5283 10.3193 20 11.4381 20H14.9527V19.9728H21.8025L16.2212 21.9421L13.7607 30.6832C13.457 31.762 14.5038 32.3999 15.46 31.7187L28.8069 22.2105C29.8438 21.4718 29.6806 20 28.5619 20H23.2321L26.2393 9.31684Z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Other Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M16.5 20H22.5"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M20.5 23L23.5 20L20.5 17"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M12 14C10.75 15.67 10 17.75 10 20C10 25.52 14.48 30 20 30C25.52 30 30 25.52 30 20C30 14.48 25.52 10 20 10C18.57 10 17.2 10.3 15.97 10.85"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Power Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M14.56 22.5599L12 19.9999L14.56 17.4399"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M17.2398 20H12.0698"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M22.2398 20H20.2798"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M26.01 14.48C27.25 15.84 28 17.71 28 20C28 25 24.42 28 20 28"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M20 12C21.05 12 22.05 12.17 22.97 12.49"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Setting Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M22.6599 28.01L21.1399 29.53C20.5199 30.15 19.4999 30.15 18.8799 29.53L17.3599 28.01C17.0999 27.75 16.5899 27.54 16.2299 27.54H14.0799C13.1999 27.54 12.4799 26.8199 12.4799 25.9399V23.79C12.4799 23.43 12.2699 22.92 12.0099 22.66L10.4899 21.14C9.8699 20.52 9.8699 19.5 10.4899 18.88L12.0099 17.36C12.2699 17.1 12.4799 16.59 12.4799 16.23V14.08C12.4799 13.2 13.1999 12.48 14.0799 12.48"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M17.3401 11.9899L18.8601 10.4699C19.4801 9.84988 20.5001 9.84988 21.1201 10.4699L22.6401 11.9899C22.9001 12.2499 23.4101 12.4599 23.7701 12.4599H25.9201C26.8001 12.4599 27.5201 13.18 27.5201 14.06V16.2099C27.5201 16.5699 27.7301 17.0799 27.9901 17.3399L29.5101 18.8599C30.1301 19.4799 30.1301 20.4999 29.5101 21.1199L27.9901 22.6399C27.7301 22.8999 27.5201 23.4099 27.5201 23.7699V25.92C27.5201 26.8 26.8001 27.5199 25.9201 27.5199"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M17 23L23 17"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M22.4945 22.5H22.5035"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
      <path
        d="M17.4945 17.5H17.5035"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Setting Two Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M23 20C23 18.34 21.66 17 20 17C18.34 17 17 18.34 17 20C17 21.66 18.34 23 20 23C20.41 23 20.81 22.92 21.17 22.76"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M14.8799 28.5799L15.9699 29.2099C16.7599 29.6799 17.7799 29.3999 18.2499 28.6099L18.3599 28.4199C19.2599 26.8499 20.7399 26.8499 21.6499 28.4199L21.7599 28.6099C22.2299 29.3999 23.2499 29.6799 24.0399 29.2099L25.7699 28.2199C26.6799 27.6999 26.9899 26.5299 26.4699 25.6299C25.5599 24.0599 26.2999 22.7799 28.1099 22.7799C29.1499 22.7799 30.0099 21.9299 30.0099 20.8799V19.1199C30.0099 18.0799 29.1599 17.2199 28.1099 17.2199C27.0999 17.2199 26.4199 16.8199 26.1799 16.1899C25.9899 15.6999 26.0699 15.0599 26.4699 14.3699C26.9899 13.4599 26.6799 12.2999 25.7699 11.7799L24.9599 11.3199"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
      <path
        d="M21.64 11.5801C20.74 13.1501 19.26 13.1501 18.35 11.5801L18.24 11.3901C17.78 10.6001 16.76 10.3201 15.97 10.7901L14.24 11.7801C13.33 12.3001 13.02 13.4701 13.54 14.3801C14.45 15.9401 13.71 17.2201 11.9 17.2201C10.86 17.2201 10 18.0701 10 19.1201V20.8801C10 21.9201 10.85 22.7801 11.9 22.7801C13.71 22.7801 14.45 24.0601 13.54 25.6301"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-miterlimit="10"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`User Icon matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <svg
      fill="none"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 40 40"
    >
      <path
        d="M20.6801 11.96C21.1601 12.67 21.4401 13.52 21.4401 14.44C21.4301 16.84 19.5401 18.79 17.1601 18.87C17.0601 18.86 16.9401 18.86 16.8301 18.87C14.4501 18.79 12.5601 16.84 12.5601 14.44C12.5601 11.99 14.5401 10 17.0001 10"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M24.4098 12C26.3498 12 27.9098 13.57 27.9098 15.5C27.9098 17.39 26.4098 18.93 24.5398 19C24.4598 18.99 24.3698 18.99 24.2798 19"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M12.1602 22.56C9.74021 24.18 9.74021 26.82 12.1602 28.43C14.9102 30.27 19.4202 30.27 22.1702 28.43C24.5902 26.81 24.5902 24.17 22.1702 22.56C19.4302 20.73 14.9202 20.73 12.1602 22.56Z"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
      <path
        d="M26.3398 28C27.0598 27.85 27.7398 27.56 28.2998 27.13C29.8598 25.96 29.8598 24.03 28.2998 22.86C27.7498 22.44 27.0798 22.16 26.3698 22"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>
</DocumentFragment>
`;
